src/ssmDocument/commands/createDocumentFromTemplate.ts

```diff
@@ -48,6 +48,8 @@ const SSMDOCUMENT_TEMPLATES: SsmDocumentTemplateQuickPickItem[] = [
 
 export async function createSsmDocumentFromTemplate(extensionContext: vscode.ExtensionContext): Promise<void> {
     let result: telemetry.Result = 'Succeeded'
+    let format: telemetry.DocumentFormat = 'Unselected'
+    let templateName: string = 'Unselected'
```

Comment:
Should this be empty string or `undefined`? Why "Unselected" ? The telemetry definition is defined as `                    "required": false` so I think `undefined` would make sense as the default.

Reply:
leaving as undefined